MCprep_addon/spawner/mcmodel.py

```diff
@@ -331,6 +331,8 @@ def add_model(
 					obj_mats.append(mat)
 					materials.append(f"#{img}")
 
+	face_dir = ["north", "south", "up", "down", "west", "east"]
```

Comment:
I would make this a tuple and place this as a constant at the top of the file (so all caps), since it's not mutated at any point in the operator.